src/ContainerCommitModal.jsx

```diff
@@ -21,7 +21,7 @@ const ContainerCommitModal = ({ container, localImages }) => {
     const [imageName, setImageName] = useState("");
     const [tag, setTag] = useState("");
     const [author, setAuthor] = useState("");
-    const [command, setCommand] = useState(container.Command ? utils.quote_cmdline(container.Command) : "");
+    const [command, setCommand] = useState(container.Config.Cmd ? utils.quote_cmdline(container.Config.Cmd) : "");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1324-20230712-150126-d59c3ba5-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)